tests/model/templates/team-response-template.constants.ts

```diff
@@ -0,0 +1,9 @@
+export const EXPECTED_TEAM_RESPONSE_HEADER = "# Team's Response";
+export const EXPECTED_TEAM_RESPONSE_TEMPLATE_CONTENT = 'This is a simple response';
+
+export const DUPLICATE_ISSUE_NUMBER = 100;
+export const SIMPLE_TEAM_RESPONSE = "# Team's Response\r\n" + 'This is a simple response\r\n' + '## Duplicate status (if any): --';
+export const TEAM_RESPONSE_WITH_WHITESPACE =
+  "# Team's Response \r\n " + '  This is a simple response\r\n ' + '## Duplicate status (if any): --';
+export const TEAM_RESPONSE_WITH_DUPLICATE =
+  "# Team's Response\r\n" + 'This is a simple response\r\n' + '## Duplicate status (if any): Duplicate of #100';
```

Comment:
Hey, thanks for updating the PR! I think the we can fix the indentation for this too,

```suggestion
export const TEAM_RESPONSE_WITH_WHITESPACE = "# Team's Response \r\n " + '  This is a simple response\r\n ' + '## Duplicate status (if any): --';
export const TEAM_RESPONSE_WITH_DUPLICATE = "# Team's Response\r\n" + 'This is a simple response\r\n' + '## Duplicate status (if any): Duplicate of #100';

```

Reply:
oops missed that out 😅